### PR TITLE
log: make zap work with go 1.6

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

File: scripts/ci/install-go.sh [hunk 1]
```diff
@@ -12,7 +12,9 @@ echo ${PATH} | grep -q "${HOME}/bin" || {
 mkdir -p ~/bin
 curl -sL -o ~/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
 chmod +x ~/bin/gimme
-eval "$(gimme 1.7)"
+
+# before changing this be sure that it will not break the RHEL packaging
+eval "$(gimme 1.6)"
```
Comment (nplanel) [inline scripts/ci/install-go.sh]: why make it running/compatible with go 1.6 ?
1.9 is almost out

Comment (nplanel) [inline scripts/ci/install-go.sh]: rpmbuild platform depend on go 1.6 ?

Comment (lebauce) [inline scripts/ci/install-go.sh]: @nplanel Indeed. CentOS/RHEL only carry Go 1.6

**Pull request merged**